parser.go

```diff
@@ -526,6 +526,19 @@ func (p *Parser) parseOption(s *parseState, name string, option *Option, canarg
 		}
 
 		err = option.Set(nil)
+	} else if option.isTerminated() {
+		args := make([]string, 0)
+		if argument != nil && *argument != "" {
+			args = append(args, *argument)
```

Comment:
Per above, this means that `--args=foo bar baz` will yield `{"foo", "bar", "baz"}`. Is that good, or do we just want to disallow `--args=` for terminated options to avoid confusion?